### PR TITLE
Clarify Scope, Out of Scope, and Model Loader API

### PR DESCRIPTION
**Pull request by anssiko**
- Add a more explicit Scope to match the CG
- Add a note to clarify the Model Loader API
- Update Out of Scope to match the CG

Based on feedback received during the AC review.

### CODE REVIEW

**Comment by dontcallmedom** [comment]
Thanks! I think the scope clarifications look reasonable - they further restrict the scope, which is safe from an IPR perspective, but we'll want to verify these restrictions are acceptable by all the interested parties.

The removal of the loader API option in the scope, and the continued uncertainty around the model format that it would need to build upon makes me wonder if there is still value in listing that API as an optional deliverable of the group - given the remaining uncertainty, it feels unlikely the group would need to start working on it in the short term, and when the time comes, it would always be an option to recharter the group to add explicitly the deliverable to the charter (if that is even needed).

**Comment by anssiko** [comment]
@dontcallmedom PR updated to remove Tentative Specifications.

**Review by dontcallmedom — APPROVED** [review]

**Comment by travisleithead** [comment]
Given: 
> Allow to setup input from various sources on the Web, e.g. array
> buffers, media streams, schedule the asynchronous hardware execution,
> and retrieve the output when hardware execution completes.

..is now noted in scope, will you be adding the various related groups into the Coordination sections?
* WebRTC (media streams) at W3C
* TC39 (array buffers) at that external org
* others I'm missing?

**Comment by anssiko** [comment]
@travisleithead thanks for your comments. @dontcallmedom, what are your thoughts on the WG Coordination section extensiveness?

The CG has been focusing on coordination and interoperability with WebAssembly, WebGPU and WebGL APIs in its incubation work and those coordinations are explicitly listed in the proposed WG Charter. Some other coordination opportunities have been discussed (e.g. WebRTC, TC39).

The [CG Coordination section](https://webmachinelearning.github.io/charter/#coordination) has a more extensive list of these coordination opportunities. We could consider bringing (some of) these over to the WG Charter.

Ecma TC39 might be a good one to be added to the External Organizations section. In addition to array buffers, TC39 has a proposal for [operator overloading in JS (Stage 1)](https://github.com/tc39/proposal-operator-overloading#matrixvector-computations) that has been [discussed](https://github.com/w3c/machine-learning-workshop/issues/73) in the recent [W3C workshop](https://www.w3.org/2020/06/machine-learning-workshop/minutes/20200923.html#slide-43) and seems helpful.

There seems to be a bit of variation on how extensively WGs list various coordinations in their charters.

**Comment by dontcallmedom** [comment]
I have added WebRTC & TC39 to the coordination section

File: charter.html [hunk 1]
```diff
@@ -428,6 +428,8 @@ <h3 id="w3c-coordination">
               Machine Learning Working Group should coordinate with this group
               to avoid overlap.
             </dd>
+            <dt><a href="https://www.w3.org/groups/wg/webrtc">WebRTC Working Group</a></dt>
+            <dd>The WebRTC Working Group defines the <code>MediaStream</code> interface and related media processing APIs that are likely make use of Machine Learning capabilities afforded by the WebNN API.</dd>
```
Comment (anssiko) [inline charter.html]:
```suggestion
            <dd>The WebRTC Working Group defines the <code>MediaStream</code> interface and related media processing APIs that likely make use of Machine Learning capabilities afforded by the WebNN API.</dd>
```

**Comment by jbingham** [comment]
After some emails with @dontcallmedom, @travisleithead and others at Microsoft, we at Google would like to keep the model loader API in scope for the charter, since we plan to work on it within the two year period and would prefer to avoid amending the charter later. From discussions already recorded in the community group, we believe Intel and Microsoft are open to it as well, which is why we'd agreed to update the charter.

Google agrees that there would need to be a standard format supported across browsers and devices. We see at least one potential path forward with ONNX, perhaps with some incremental evolution of the op set.

**Comment by dontcallmedom** [comment]
@travisleithead @jbingham based on this, I've re-integrated the ML loader API in the scope of the charter and clarified what the dependency for that work item should be:
> The Model Loader API needs a standard format supported across browsers and devices for broad interoperability. The Working Group will only start working on this API if such a format emerges.

Does that look right? If not, would you mind suggesting an alternative approach to represent what we want the group to achieve?

**Comment by jbingham** [comment]
I think that wording is probably fine. What I take it to mean: the Community Group can work on this API, and incubate, and implement based on a best guess as to what the format could be. The Working Group can move it forward as a web standard once there is agreement about the format. Is that right?

**Comment by dontcallmedom** [comment]
@jbingham correct

**Comment by travisleithead** [comment]
I hope I may suggest one more fine-tuning of the note about the Model Loader API from internal feedback. I think this also incorporates a bit of what @jbingham [was noting](https://github.com/w3c/machine-learning-charter/pull/15#issuecomment-809728866):

> Note: The Model Loader API needs a standard format supported across browsers and devices for broad interoperability. The Working Group will only create a standard format of its own if no other standardized format aligns with the group's principles nor allows the group to control or provide direct input to shape it. The Working Group will only start working on this API when there is agreement on such a format.

With these changes, we are happy with the rest of the charter updates.

**Comment by dontcallmedom** [comment]
@jbingham I've [pushed another commit](https://github.com/w3c/machine-learning-charter/pull/15/commits/67766302ffad5155bce59f944381687bdbd446ef) incorporating @travisleithead 's suggestion - can you confirm you're ok with that addition?

**Comment by jbingham** [comment]
lgtm

**Pull request merged**